cmd/tether/attach.go

```diff
@@ -289,7 +293,9 @@ func backchannel(ctx context.Context, conn net.Conn) error {
 	go func() {
 		select {
 		case <-ctx.Done():
-			conn.Close()
+			if conn != nil {
```

Comment:
If line `274` is true, how does conn go to `nil` here?  And if it can go to `nil`, how do you avoid it going to `nil` on 297 after the check on 296?